packages/app-store/googlecalendar/api/add.ts

```diff
@@ -9,6 +9,7 @@ import { encodeOAuthState } from "../../_utils/oauth/encodeOAuthState";
 const scopes = [
   "https://www.googleapis.com/auth/calendar.readonly",
   "https://www.googleapis.com/auth/calendar.events",
+  "https://www.googleapis.com/auth/userinfo.email",
```

Comment:
adding scope to get email access from google account

Reply:
`note`: AFAIK adding this scope does not change the permissions we require from Google, but worth checking, cc @keithwillcode @hariombalhara

Reply:
Tagging @joeauyeung as well so we confirm this is what we want

Reply:
@exception see this comment about adding to our current scope
https://github.com/calcom/cal.com/pull/11639/files#r1362753492